sdk/stats.js

```diff
@@ -22,8 +22,7 @@ const stats = async function stats(apiKey, since, until) { // eslint-disable-lin
   };
 
   logger.debug('Fetching account usage stats', { statsOptions });
-  const metrics = await loggedRequest(statsOptions, 'get');
-  return metrics;
+  return getValidatedBody(statsOptions, 'get');
```

Comment:
why drop the `await`?
it doesn't matter right now, but it's a matter of style and design.
if `getValidatedBody` errors should be handled by `stats` (or maybe augmented) it should have an `await`, otherwise, if it's the caller responsibility, a simple `return` is in place

Reply:
Well it's not in a try/catch so the await didn't matter anyway. I'm also assuming that's why linting failed (because the await doesn't matter here).